OADP-2939: Release notes OADP 1.2.3 (#66512)
openshift/openshift-docs: modules/oadp-release-notes-1-2-0.adoc

[id="new-features_{context}"]
== New features

.Resource timeouts

> The new `resourceTimeout` option specifies the timeout duration in minutes for waiting on various Velero resources. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. The default duration is ten minutes.

Reviewer: ISG nit: use numerals to represent all numbers
```suggestion
The new `resourceTimeout` option specifies the timeout duration in minutes for waiting on various Velero resources. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. The default duration is 10 minutes.
```

[id="known-issues-1-2-0_{context}"]
== Known issues

The following issues have been highlighted as known issues in the release of OADP 1.2.0:

.Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack)

> The HTTP/2 protocol allows a denial of service (server resource consumption) because request cancellation can reset multiple streams quickly. The server has to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection, which results in a denial of service due to server resource consumption. For a list of all OADP issues associated with this CVE, see the following link:https://issues.redhat.com/browse/OADP-2868?filter=12421248[Jira list].

Reviewer: ISG says to avoid parentheses in general text. Also, suggestion to break up the long sentence. So maybe something like:
```suggestion
The HTTP/2 protocol is susceptible to a denial of service attack because request cancellation can reset multiple streams quickly. The server has to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection. This results in a denial of service due to server resource consumption. For a list of all OADP issues associated with this CVE, see the following link:https://issues.redhat.com/browse/OADP-2868?filter=12421248[Jira list].
```
since you actually specify the attack later as server resource consumption, I think it makes sense to omit that from the beginning.

also, I can't see the Jira list unless i'm logged in. that may be expected, but just figured i'd note it here too.

Author: Nice... really like `... is susceptible to... `

@jldohmann - please can I ask your advice on these Jiras: everything in the Jira list is set to the security list `RH employee or consultant`, so should we include the Jira list? I can see pros and cons, but I remain undecided?

Thanks
